Mathlib/Algebra/GroupWithZero/WithZero.lean

```diff
@@ -266,6 +266,10 @@ def unitsWithZeroEquiv : (WithZero α)ˣ ≃* α where
   right_inv _ := rfl
   map_mul' _ _ := coe_inj.mp <| by simp only [Units.val_mul, coe_unzero, coe_mul]
 
+theorem units_val_eq_coe_unitsWithZeroEquiv (γ : (WithZero α)ˣ) :
+    γ.val = ↑(unitsWithZeroEquiv γ) := by
+  simp only [WithZero.unitsWithZeroEquiv, MulEquiv.coe_mk, Equiv.coe_fn_mk, WithZero.coe_unzero]
```

Comment:
Usually the more complicated expression goes on the left hand side, because then you can simplify expressions by rewriting with this lemma from left to right.
Could you please swap the two sides of the equation (and the name)?